reledmac.dtx

```diff
@@ -1725,11 +1725,11 @@
 % \pend
 % \endnumbering
 % \end{ledgroup}
-% Here, it is not ambiguous what \enquote{per causam} refers to.
+% In this case it is clear which \enquote{per causam} is meant.
 %
-% However, we may think that as \enquote{per causam} is the lemma of the second note, there should be only one number for the whole lemma.
+% However, it may happen that \enquote{per causam} is the lemma of the second note, and there should be only one number for the whole lemma.
```

Comment:
Now, it is really "we may think". If you consider the whole lemma "per causam", you want to have only one number. If you consider individual words "per" and "causam" which form the lemma, you want to have one number for "per" and another one for "causam".